frontend/src/core/hooks/useStatus.js

```diff
@@ -0,0 +1,89 @@
+import { useMutation } from "react-query";
+import { useToast } from "./";
+import { AuthService } from "../../core/services";
+
+const useStatus = () => {
+	const toast = useToast();
+
+	const { mutate: apiServerStatus, data: apiServerData } = useMutation(
+		AuthService.apiServerStatus,
+		{
+			onError: (error) => {
+				// toast(error, "error");
```

Comment:
Get rid of these handlers if they are not doing anything. Definitely get rid of these comments.